src/Missings.jl

```diff
@@ -213,11 +213,32 @@ struct SpreadMissings{F} <: Function
 end
 
 function (f::SpreadMissings{F})(xs...; kwargs...) where {F}
+
     if any(x -> x isa AbstractVector{>:Missing}, xs)
         vecs = Base.filter(x -> x isa AbstractVector, xs)
 
         findex = eachindex(first(vecs))
-        @assert all(x -> eachindex(x) == findex, vecs)
+        if !(all(x -> eachindex(x) == findex, vecs[2:end]))
```

Comment:
```suggestion
        if !(all(x -> eachindex(x) == findex, view(vecs, 2:end)))
```

Reply:
`vecs` is a small tuple, here. So view doesn;t work.